source/dofs/dof_handler_policy.cc

```diff
@@ -3013,9 +3013,13 @@ namespace internal
             // know how to serialize itself. consequently, first copy it over
             // to an array of bytes, and then serialize that
             std::vector<char> quadrants_as_chars (sizeof(quadrants[0]) * quadrants.size());
-            std::memcpy(quadrants_as_chars.data(),
-                        quadrants.data(),
-                        quadrants_as_chars.size());
+            if (quadrants_as_chars.size()>0)
+              {
+                Assert(quadrants.data() != nullptr, ExcInternalError());
```

Comment:
same here and probably elsewhere

Reply:
Although here you use `data()`, and that is guaranteed to return a pointer. So there is no harm in the assertion -- I just can't see it to ever by useful.

Reply:
I'll add this in a following PR.